clang/test/CodeGen/AArch64/fp8-intrinsics/acle_neon_fp8_cvt.c

```diff
@@ -330,3 +330,6 @@ mfloat8x8_t test_vcvt_mf8_f16_fpm(float16x4_t vn, float16x4_t vm, fpm_t fpm) {
 mfloat8x16_t test_vcvtq_mf8_f16_fpm(float16x8_t vn, float16x8_t vm, fpm_t fpm) {
   return vcvtq_mf8_f16_fpm(vn, vm, fpm);
 }
+
+// CHECK: declare void @llvm.aarch64.set.fpmr(i64)  [[ATTR1:#.*]]
+// CHECK: attributes [[ATTR1]] = { nocallback nofree nosync nounwind willreturn memory(inaccessiblemem: write) }
```

Comment:
The test changes look manual? Which is going to make it awkward in the future.  Perhaps add a dedicated test file for this specific purpose?

Reply:
I've tried to find similar test in Sema, but I think what I want is just when we have the llvm-ir/codegen.
Let me know if the test I added is fine. 
It is also now checking other fp8  functions memory attributes, besides set.fpmr.